site/assets/scss/app.scss

```diff
@@ -982,3 +982,62 @@ a.classifiation-tag:hover {
     }
   }
 }
+
+// Notification Dropdown
+.notify-drop-footer {
+  border-top: 2px solid #e2e2e2;
+}
+.notify-drop {
+  min-width: 330px;
+  background-color: #fff;
+  min-height: 280px;
+  max-height: 280px;
+}
+.notify-drop .notify-drop-title {
+  border-bottom: 2px solid #e2e2e2;
+  padding: 5px 15px 10px 15px;
+}
+.notify-drop .notify-drop-title .notification-title {
+  font-weight: bold;
+}
+.notify-drop .drop-content {
+  min-height: 200px;
+  max-height: 200px;
+  overflow-y: scroll;
+}
+.notify-drop .drop-content::-webkit-scrollbar-track {
+  background-color: #F5F5F5;
+}
+.notify-drop .drop-content::-webkit-scrollbar {
+  width: 8px;
+  background-color: #F5F5F5;
+}
+.notify-drop .drop-content::-webkit-scrollbar-thumb {
+  background-color: #ccc;
+}
+
+// Notification List
+.notification .unseen {
+  padding: 10px 0 5px 0;
+  background-color: #eeeeee;
+}
+.notification .seen {
+  padding-left: 15px;
+}
+.notification .date {
+  font-size: 10px;
+  display: inline-block;
+  color: #B5B6B4;
+  padding:0;
+}
+.notification-markasseen {
+  background-color: transparent;
+  border: 0;
+}
+.notification-remove {
+  background-color: transparent;
+  border: 0;
+}
+.notification .markasseenIcon:hover {
```

Comment:
We don't use lowerCamelCase in CSS classes, we use dashes. So this should be .markasseen-icon